### PR TITLE
Updated dashboard.html with wishlist

### PR DESCRIPTION
**Pull request by Andreassim**
Added new functionality to WishLisControler.
Here we can now add a new wishlist for a loggedin user.
Moved dashboard in here

WishlistRepository updated wist a method for retrieve all wishlists for a given user.

Wishlist service updated to have the update from repository

Nice fronted :))))

What kind of change does this PR introduce?

- [ ] `bug-fix`
- [x] `new-feature`
- [x] `enhancement`

### CODE REVIEW

File: src/main/java/com/group6/wishshare/web/WishlistController.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+package com.group6.wishshare.web;
+
+import com.group6.wishshare.domain.model.User;
```
Comment (MathiasReker) [inline src/main/java/com/group6/wishshare/web/WishlistController.java:3]: Use a service in front of the model

File: src/main/java/com/group6/wishshare/data/repository/WishListRepository.java [hunk 2]
```diff
@@ -65,4 +68,27 @@ public Wishlist getWishlist(int id) {
     }
     return null;
   }
+
+  public List<Wishlist> getWishLists(int user_id) {
+    String stm = "SELECT * FROM wishlist WHERE user_id = ?";
+    PreparedStatement prep;
```
Comment (MathiasReker) [inline src/main/java/com/group6/wishshare/data/repository/WishListRepository.java:74]: Can we use `ps` everywhere?

**Review by MathiasReker — CHANGES_REQUESTED** [review]

File: src/main/java/com/group6/wishshare/data/repository/WishListRepository.java (hunk 2 above)
Comment (MathiasReker) [reply src/main/java/com/group6/wishshare/data/repository/WishListRepository.java:74]: and move the line inside the try block :)

File: src/main/java/com/group6/wishshare/web/WishlistController.java (hunk 1 above)
Comment (Andreassim) [reply src/main/java/com/group6/wishshare/web/WishlistController.java:3]: we need a userservice class for this then. FrontController does the same.

Not fixing this in this PR

**Review by Jarkyman — APPROVED** [review]

File: src/main/java/com/group6/wishshare/data/repository/WishListRepository.java (hunk 2 above)
Comment (Andreassim) [reply src/main/java/com/group6/wishshare/data/repository/WishListRepository.java]: fixed

**Pull request merged**